javascript2/week1/optional-homework.md

```diff
@@ -1,11 +1,7 @@
 > [!WARNING]
-> You do not need to complete these homeworks. We are currently testing new homeworks for JS2 and JS3. More info [here](https://github.com/HackYourFuture-CPH/js-projects).
+> These are optional homework exercises that you can complete on top of your [homework project](/homework-projects/readme.md), if you wish. These will not be reviewed.
```

Comment:
when I'm a reviewer I usually don't mind looking through those additional homeworks. Maybe `These will not be reviewed` is a little strong? Maybe `These are not required to be reviewed, although feel free to ask your mentor to do so`? Or something along these lines... Some students are very word to word :)

Reply:
That's nice to hear, yes I'll change the wording along the lines you suggest :)